js/src/components/RectangleGraphNode.vue

```diff
@@ -117,6 +129,14 @@ export default class RectangleGraphNode extends Vue {
     });
   }
 
+  // measure text width in pixels
+  measureText(text) {
```

Comment:
I can see that the EllipseGraphNode.vue class contains the same function. Perhaps we could create a base class or there is one already where we can add common functions.

Reply:
will do in the future if more common functions between the two vue